src/utils/components/MemorySizeInput/MemorySizeInput.tsx

```diff
@@ -98,4 +97,4 @@ const DiskSizeNumberInput: React.FC<DiskSizeNumberInputProps> = ({ diskSize, onC
   );
 };
 
-export default DiskSizeNumberInput;
+export default MemorySizeInput;
```

Comment:
in a quick look, i can see this component can be used a memory, CPU, PVC, and migration sizing, so memoryinput as a name cant work for all cases, need a more general name, in my opinion.
This component used PVC_SIZE_FORMAT I think this should be changed to a more appropriate name? maybe also moved? The same goes for fieldID prop, which is hardcoded for PVC

Reply:
This component is only set to handle memory-type units as-is. It would need to be refactored to handle additional unit types. Is that what you're suggesting?

Reply:
as I see it, this component is handling input number with a dropdown for unit TIB, GIB, MIB, which can be used for disk size, memory size, etc.. so calling it only memory isn't really matching the usage for disk size or others... The same goes for fieldId and PVC_SIZE_FORMAT, as it isn't only PVC. should be something like UNITS_FORMATS

Reply:
I'm a bit confused because your original comment said it could be used for CPUs as well. So, just to clarify, you do mean to leave it where it works anywhere where base 2 capacity units (MiB, GiB, TiB, etc) are being used, correct? 

If so, I suggest `CapacityInput`.

Reply:
CapacityInput is cool, just make sure to take care of everything mention with pvc-* in this file :)

Reply:
I think I've already done that part. I just haven't pushed it yet. =)